### PR TITLE
Use float division in calculating PageRank and add tests

### LINKED ISSUE #107: [5] Incremental Page Rank returns values above 1
Sometimes, the incremental PageRank algorithm makes mistakes and returns values above 1.

- [x] Find the bug
- [x] Fix the bug

### PR DESCRIPTION
**Pull request by vandenheuvel**
Fixes #107.

I added tests for our PageRank implementation and fixed a division bug.

### CODE REVIEW

File: Tribler/Test/Community/Multichain/statistics/test_page_rank.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        nr_of_nodes = 1000
+        nr_of_edges = 100000
+        self.graph = gnm_random_graph(nr_of_nodes, nr_of_edges)
+        self.page_rank = IncrementalPageRank(self.graph)
+
+    def test_max_min(self):
+        self.page_rank.initial_walk()
+        self.page_rank.count()
+        ranks = self.page_rank.get_ranks()
+        self.assertTrue(all(0 <= rank <= 1 for rank in ranks.values()))
```
Comment (yinghaodai) [inline Tribler/Test/Community/Multichain/statistics/test_page_rank.py]: Nice!

File: Tribler/Test/Community/Multichain/statistics/test_page_rank.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+from Tribler.community.multichain.statistics.page_rank import IncrementalPageRank
+
+
+class TestIncrementalPageRank(TestCase):
+    """
+    As it is a Monte Carlo method, integration tests can only be basic.
+    """
+    def setUp(self):
+        nr_of_nodes = 1000
```
Comment (rkeulemans) [inline Tribler/Test/Community/Multichain/statistics/test_page_rank.py]: This look like some magic numbers to me. Think about a config for the test cases.

File: Tribler/Test/Community/Multichain/statistics/test_page_rank.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        nr_of_edges = 100000
+        self.graph = gnm_random_graph(nr_of_nodes, nr_of_edges)
+        self.page_rank = IncrementalPageRank(self.graph)
+
+    def test_max_min(self):
+        self.page_rank.initial_walk()
+        self.page_rank.count()
+        ranks = self.page_rank.get_ranks()
+        self.assertTrue(all(0 <= rank <= 1 for rank in ranks.values()))
+        self.assertAlmostEqual(sum(ranks.values()), 1, delta=0.05)
```
Comment (rkeulemans) [inline Tribler/Test/Community/Multichain/statistics/test_page_rank.py]: Magic number again.

Comment (vandenheuvel) [inline Tribler/Test/Community/Multichain/statistics/test_page_rank.py]: I choose to give a name to this variable.

**Pull request merged**